components/servo/lib.rs

```diff
@@ -237,12 +237,8 @@ fn create_constellation(opts: opts::Opts,
                         layout_thread::LayoutThread,
                         script::script_thread::ScriptThread>::start(initial_state);
 
-    // Send the URL command to the constellation.
```

Comment:
Because the code is already clear enough to be self-explanatory, IMHO. [A comment is an apology](http://butunclebob.com/ArticleS.TimOttinger.ApologizeIncode).